howso/engine/trainee.py

```diff
@@ -235,35 +252,31 @@ def save_location(self) -> Union[Path, str, None]:
                 return None
 
     @property
-    def name(self) -> Optional[str]:
+    def name(self) -> str | None:
         """
         The name of the trainee.
 
         Returns
         -------
-        str or None
+        str | None
             The name.
         """
         return self._name
 
     @name.setter
-    def name(self, name: Optional[str]) -> None:
+    def name(self, name: str | None):
         """
         Set the name of the trainee.
 
         Parameters
         ----------
-        name : str or None
+        name : str | None
             The name.
-
-        Returns
-        -------
-        None
         """
         if name is not None and len(name) > 128:
             raise ValueError(
-                "Invalid value for `name`, length must be less "
-                "than or equal to `128`"
+                "Invalid value for ``name``, length must be less "
```

Comment:
Can we talk about this? Double backticks is a sphinx thing. Literal strings emitted for the customer on the command line won't be parsed with Sphinx and you're establishing a style that will be very different from the rest of the code base here.

Reply:
This is an unfortunate victim of my single-but-not-double-backticks finding regex. Nice catch.